### PR TITLE
Improved PID values

### PR DESCRIPTION
**Pull request by glopezdiest**
Moved the **hardcoded PID values into functions** (_setLaneFollowingPIDControllers_ was already there, added _setTurnPIDControllers_, _setLaneChangingPIDControllers_). Their **values have also been updated** and some vehicle blueprint have been uncommented as their PID values are now much better. It is important to note that PR #11 has changes regarding the synchronous mode which makes the PID values perform much much better.

Additonally, moved `RegulatedControlAction(throttle, current_steer_angle, past_steer_angle)` at `FollowLaneBehavior` to several lines before, as it was being done after saving the values of that frame into its past variables

### CODE REVIEW

File: src/scenic/simulators/carla/blueprints.scenic [hunk 1]
```diff
@@ -7,15 +7,15 @@ carModels = {# for speed = 10
       'vehicle.audi.etron', # K_P=0.3, K_D=0.2, K_I=0
       'vehicle.audi.tt',
       'vehicle.bmw.grandtourer',
-      # 'vehicle.bmw.isetta', # current PID controller not work 
+      'vehicle.bmw.isetta', # current PID controller not work
```
Comment (dfremont) [inline src/scenic/simulators/carla/blueprints.scenic:10]: Probably should remove the comment if it's not accurate anymore. Ditto for "wrangler_rubicon" below.

File: src/scenic/domains/driving/behaviors.scenic [hunk 2]
```diff
@@ -26,6 +26,28 @@ def setLaneFollowingPIDControllers(is_vehicle, dt):
 
     return lon_controller, lat_controller
 
+def setTurnPIDControllers(is_vehicle, dt):
```
Comment (dfremont) [inline src/scenic/domains/driving/behaviors.scenic:29]: These new functions don't seem to get used. Is part of the PR missing?

**Review by dfremont — CHANGES_REQUESTED** [review]
Thanks for these improvements and fixes! It looks like the new PID functions didn't get hooked up properly though (maybe a commit is missing?) - see comments below.

**Comment by ek65** [comment]
The same controller issue that I aforementioned in "CARLA scenarios #11" PR persists here.

File: src/scenic/domains/driving/behaviors.scenic (hunk 2 above)
Comment (glopezdiest) [reply src/scenic/domains/driving/behaviors.scenic:29]: Indeed, there was one commit missing...

File: src/scenic/simulators/carla/blueprints.scenic (hunk 1 above)
Comment (glopezdiest) [reply src/scenic/simulators/carla/blueprints.scenic:10]: Done

**Comment by glopezdiest** [comment]
Added a missing commit. The PIDs are still flawed, and will overshoot at closed turns, but this should fix the majority of the cases. I think that in order to fix it, the lateral controller should be changed from checking the distance to the center line, to tracking a waypoint in front.

File: src/scenic/simulators/carla/blueprints.scenic (hunk 1 above)
Comment (dfremont) [reply src/scenic/simulators/carla/blueprints.scenic:10]: Comments are still there - maybe another commit is missing?

File: src/scenic/domains/driving/behaviors.scenic (hunk 2 above)
Comment (dfremont) [reply src/scenic/domains/driving/behaviors.scenic:29]: Oops, now there are two copies of `setTurnPIDControllers` and `setLaneChangingPIDControllers`, and they have different numerical parameters so I'm not sure which one is correct. Can you please fix?

**Review by dfremont — CHANGES_REQUESTED** [review]
Nearly there, but a couple functions got duplicated and I'm not sure how to fix it - please see the comments.

**Comment by glopezdiest** [comment]
Resolved the conflicts and removed the comments (now for sure)

**Comment by dfremont** [comment]
Hmm, I'm still seeing two copies of `setTurnPIDControllers` and `setLaneChangingPIDControllers`. I also can't see my comment about them, so maybe I submitted the review wrong.

**Comment by glopezdiest** [comment]
Indeed, removed duplicates

**Review by dfremont — APPROVED** [review]

**Pull request merged**